openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -223,6 +246,30 @@
                     updateButton();
                 });
 
+            // tree operator label
+            nodeEnter.append("text")
+                .text(function(d) {
+                    if (d.children){
+                        var operator = d.operator? d.operator : DEFAULT_OPERATOR;
+                        d.operator = operator;
+                        return operator;
+                    }
+                })
+                .style("fill", function(d) {
+                    if (d.operator != undefined) {
```

Comment:
My editor suggests to use `!==` instead of `!=` but I am always quite confused about those things in javascript.